content/ansible/roles/manageiq-core.manageiq-automate/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+galaxy_info:
+  author: "Drew Bomhof"
+  company: Red Hat, Inc.
+  license: license (Apache)
+  min_ansible_version: 2.2
+  platforms:
+   - name: EL
+     versions:
+      - 7
+
+   - name: Fedora
+     versions:
+      - 25
+      - 26
+   - name: Ubuntu
+     versions:
+      - xenial
+      - precise
+  galaxy_tags: [manageiq, cloudforms, automate, rhel]
```

Comment:
@syncrou Why have the galaxy stuff in here?  If we keep it, I think we should

- remove cloudforms tag
- change author to "ManageIQ Developers" as we do for the [rest of the manageiq-content](https://github.com/ManageIQ/manageiq-content/blob/master/manageiq-content.gemspec#L10).

Are the platforms necessary?  I'd hate to have to maintain that list.

Reply:
Maybe I can pull the `galaxy_tags:` for this role - it obviously was copied and pasted from Galaxy.